connectors/sources/github.py

```diff
@@ -1207,7 +1332,7 @@ async def _remote_validation(self):
             msg = "Configured token does not have required rights to fetch the content. Required scopes are 'repo', 'user', and 'read:org'."
             raise ConfigurableFieldValueError(msg)
 
-        if self.github_client.repos != [WILDCARD]:
+        if WILDCARD not in self.configured_repos:
```

Comment:
Ha good catch!

Reply:
It's just a personal preference, to handle an edge case, when a customer configures the `repositories` like `elastic/kibana, *, elastic/connectors`.

In this case, we will just sync all repos from the org.